pkg/api/handlers/compat/version.go

```diff
@@ -46,22 +47,25 @@ func VersionHandler(w http.ResponseWriter, r *http.Request) {
 		},
 	}}
 
+	apiVersion := utils.APIVersion[utils.CompatTree][utils.CurrentAPIVersion]
+	minVersion := utils.APIVersion[utils.CompatTree][utils.MinimalAPIVersion]
+
 	utils.WriteResponse(w, http.StatusOK, entities.ComponentVersion{
 		Version: docker.Version{
 			Platform: struct {
 				Name string
 			}{
 				Name: fmt.Sprintf("%s/%s/%s-%s", goRuntime.GOOS, goRuntime.GOARCH, infoData.Host.Distribution.Distribution, infoData.Host.Distribution.Version),
 			},
-			APIVersion:    components[0].Details["APIVersion"],
+			APIVersion:    fmt.Sprintf("%d.%d", apiVersion.Major, apiVersion.Minor),
```

Comment:
Are we sure there is a separate Libpod handler for this? I can't find one.

Reply:
There is one handler.  For the Libpod values, you need to look for the "Podman Engine" component. The non-component  values are for older versions of the Docker API.